solidity/test/TBTCSystemTest.js

```diff
@@ -336,6 +336,14 @@ describe("TBTCSystem", async function() {
       expect(allowNewDeposits).to.equal(false)
     })
 
+    it("pauses new deposit creation after the 1-day grace period", async () => {
+      await increaseTime(24 * 60 * 60 + 1)
+      await tbtcSystem.emergencyPauseNewDeposits()
+
+      const allowNewDeposits = await tbtcSystem.getAllowNewDeposits()
+      expect(allowNewDeposits).to.equal(false)
```

Comment:
I’m confused... what is this checking? Don’t fully understand the grace period either, I think —the PR description doesn’t really explain what it is meant to do practically, just something about “ramping up” for signers?

Reply:
It began as a 0 supply cap, which meant `getAllowNewDeposits()` was false. It also had the side-effect of breaking a bunch of our tests.

I updated the tests, then realized a 1 BTC limit was actually more appropriate for day 1.

Reply:
Adding.. more tests here. I want to show that adding the supply cap doesn't restrict the emergency pause.